webpack/extend/CardItem.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable camelcase */
+
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Skeleton from 'react-loading-skeleton';
+import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  OkIcon,
+  ErrorCircleOIcon,
+} from '@patternfly/react-icons/dist/js/icons';
+import {
+  PropertiesSidePanel,
+  PropertyItem,
+} from '@patternfly/react-catalog-view-extension';
+
+import { get } from 'foremanReact/redux/API';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import CardItem from 'foremanReact/components/common/CardTemplate/CardItem';
+import RelativeDateTime from 'foremanReact/components/common/dates/RelativeDateTime';
+
+const ExtendedCardItem = ({ hostDetails: { name } }) => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (name) {
+      dispatch(
+        get({
+          key: 'HOST_DETAILS_JOBS',
+          url: `/api/job_invocations?search=host%3D${name}&per_page=3`,
+        })
+      );
+    }
+  }, [dispatch, name]);
+  const jobs = useSelector((state) =>
+    selectAPIResponse(state, 'HOST_DETAILS_JOBS')
+  );
+
+  const iconMarkup = (status) => {
+    if (status === 1) return <ErrorCircleOIcon style={{ color: 'red' }} />;
+    return <OkIcon style={{ color: 'green' }} />;
+  };
+
+  return (
+    <CardItem header="Recent Jobs">
+      <PropertiesSidePanel>
+        {jobs?.results?.map(
+          ({ status, dynflow_task: { id }, start_at, description }) => (
+            <PropertyItem
+              label={
+                description ? (
+                  <Grid>
+                    <GridItem span={10}>
+                      <ElipsisWithTooltip>{description}</ElipsisWithTooltip>
+                    </GridItem>
+                    <GridItem span={2}>{iconMarkup(status)}</GridItem>
+                  </Grid>
+                ) : (
+                  <Skeleton />
+                )
+              }
+              value={
+                start_at ? (
+                  <a href={`/foreman_tasks/tasks/${id}`}>
```

Comment:
I think we should rather link job invocation detail page than the task itself. Task page is more for debugging purposes. Also I'm thinking the link should probably be rather the job description than the date, but not a strong opinion on this.

Reply:
and one more thing, how do we handle permsisions? We should display the card only if user has view_job_invocations permission. Or we can always display it and only authorize the link (does a user have a permission to view the specific job).

Reply:
The data is fetched atm via API, I guess if a user doesn't have permission for  `view_job_invocations`,  the given data will be empty, right? If so we can render an empty state, or not render it at all.